android/README

```diff
@@ -38,7 +38,7 @@ $ gradle build
 With an Android emulator running or an Android device connected, the following
 command line then builds the library and runs the tests:
 
-$ gradle connectedAndroidTest
```

Comment:
I wonder why this change was made, internally it says (correctly) connectedAndroidTest already, this shouldn't be reverted